JamitFoundationExample/App/Sources/SwiftUISupport/UIKitIntegrationView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  SwiftUISampleView.swift
+//  JamitFoundationExample
+//
+//  Created by Murat Yilmaz on 20.06.22.
+//  Copyright © 2022 Jamit Labs GmbH. All rights reserved.
+//
+
+import Foundation
+import JamitFoundation
+import SwiftUI
+import SwiftUISupport
+import UIKit
+
+@available(iOS 13.0, *)
+struct UIKitIntegrationView: View {
+    @State private var images: [URL] = []
+
+    var body: some View {
+        ScrollView {
+            VStack {
+                StatefulViewRepresentable<ActionView<ListItemView>>(
+                    model: .constant(
+                        ActionViewModel(
+                            content: ListItemViewModel(title: "Add"),
+                            action: {
+                                images.append(URL(string: "https://picsum.photos/200")!)
+                            }
+                        )
+                    )
+                )
+                .frame(height: 44)
+
+                ForEach(images, id: \.self) { url in
```

Comment:
When running this, I got a warning

```
ForEach<Array<URL>, URL, ModifiedContent<StatefulViewRepresentable<ImageView>, _FrameLayout>>: the ID https://picsum.photos/200 occurs multiple times within the collection, this will give undefined results!
```

While this is just a warning now, maybe it will lead to actual problems with future SwiftUI updates. So I'd suggest to change the line to

```swift
ForEach(Array(images.enumerated()), id: \.offset) { _, url in
```